boskos/README.md

```diff
@@ -0,0 +1,51 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
```

Comment:
Grossly over designed. CloudSQL adds an immense amount of weight. We have no needs here that a json object cannot handle.

Likewise it is not clear to me that the weight of adding a PD to the deployment and the restrictions it comes with is worth it over serializing/deserilizing the json blob to GCS (which would have marginally more complex logic in the code).